index.yaml

```diff
@@ -100,3 +100,9 @@ sources:
     license: GPLv3
     url: https://raw.githubusercontent.com/travisbgreen/hunting-rules/master/hunting.rules
     min-version: 4.1.0
+
+versions:
+
```

Comment:
nit: Extra line is probably not required here.

Reply:
Fixed during merge.